### PR TITLE
notes on how to use the JWT create, decode and verify

### PR DESCRIPTION
**Pull request by eeerin**

### CODE REVIEW

File: docs/workflows/logic/jwt-create.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+* ES512
+
+
+## Outputs
+
+The JWT create node has one optional output field. The `Destination Path` field takes a [payload path](/workflows/accessing-payload-data/#payload-paths). If it is defined and the JWT successfully created, it will be added to your payload at said path, otherwise an error object will be there.
+
+## Options
+
+If any of these options are set in JSON template, in the data source object they will **not** be overwritten by these options.
```
Comment (spacetc62) [inline docs/workflows/logic/jwt-create.md]: wording here is a bit weird

File: docs/workflows/logic/jwt-create.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# JWT Create
+
+The JWT Create node allows a workflow to create a JSON Web Token from a json object and secret and places it on the payload.
```
Comment (spacetc62) [inline docs/workflows/logic/jwt-create.md]: link to the JSON web token spec?

File: docs/workflows/logic/jwt-create.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+![JWT Create Node](/images/workflows/logic/jwt-create-node.png "JWT Create Node")
+
+## Configuration
+
+JWT Create node configuration requires at least a `JSON Template` object, `Secret Template` and an `Algorithm Type`. The JSON template can be given as a path from the payload or a JSON template object, both should resolve to an object. The secret template can either be a payload template path, or the secret itself. The default alogithm type is `HS256`, see the list below for all algorithms.
```
Comment (anaptfox) [inline docs/workflows/logic/jwt-create.md]: Could mention here that it could be a secret or private key

**Review by spacetc62 — APPROVED** [review]

File: docs/workflows/logic/jwt-create.md (hunk 3 above)
Comment (eeerin) [reply docs/workflows/logic/jwt-create.md]: It seems like secret key and private key are synonyms in the crypto world. http://searchsecurity.techtarget.com/definition/private-key

I'll add key after Secret every where and I think that should help.

**Pull request merged**